scripts/alignment_score.py

```diff
@@ -0,0 +1,110 @@
+import sys
+import os
+
+def scrape_input(path: str, delim: str = " ", get_nodes: bool = False)-> list: 
+    file = open(path, "r")
+    lines = file.readlines()
+    rlist = []
+    for line in lines:
+        if get_nodes:
+            rlist.extend(
+                line.strip("\n").split(delim)[1:] # get all the nodes and discard the index value
+            )
+        else:
+            rlist.append(
+                line.strip("\n").split(delim)[0] # only get index value
+            )
+    return rlist
+
+def alignment_score(indexA: list, indexB: list, match: int, miss: int, gap: int):
+    matrix = [[0]*(len(indexA) + 1) for i in range(0, len(indexB) + 1)] # empty matrix
+    rowval = [x for x in indexB]
+    rowval.insert(0, -1)
+    colval = [x for x in indexA]

```

Comment:
super small thing, can you also rename indexA and indexB to index_a and index_b?

Reply:
oh yea forgot to do that